ansible_rulebook/engine.py

```diff
@@ -285,10 +284,6 @@ async def run_actions(self, data: Dict):
                 )
                 result = await self.call_action(*item)
 
-            # create a new plan queue for current event so that results
-            # can be concatenated
-            self.ruleset_queue_plan.plan.queue = asyncio.Queue()
```

Comment:
I did some tests. Only these three lines can be removed while other changes are not needed.

Reply:
@bzwei Updated please review